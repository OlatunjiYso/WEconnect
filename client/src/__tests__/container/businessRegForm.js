import React from "react";
import { shallow } from 'enzyme';

import BusinessRegForm from '../../containers/business_reg_form';

describe('All tests for business reg container', () => {
  
  const initialState = {};

  const store = mockStore(initialState);
  const wrapper = shallow(<Provider store={store}><BusinessRegForm/></Provider>);
  describe('it should render self', ()=>{
    it('it should render businessReg page', () => {
      expect(wrapper.length).toEqual(1);
    });
    it('should match snapshot', () => {
      expect(wrapper).toMatchSnapshot();
    });
  }); 
});
