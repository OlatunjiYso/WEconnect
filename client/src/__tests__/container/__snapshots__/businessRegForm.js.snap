// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`All tests for business reg container it should render self should match snapshot 1`] = `<Connect(BusinessRegForm) />`;
